src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -136,42 +132,42 @@ object StreamingAOIMonitor
                 .map(s => Map(Source.EndSequence -> s.toString))
                 .getOrElse(Map.empty[String, String])
 
-            val aoiIndex = getAreaIndex( /*databaseUri*/ )
             val aoiTag = udf { g: jts.Geometry =>
               aoiIndex(g).toList
             }
+            val notificationId = udf { n: NotificationData =>
```

Comment:
We need to pull the notification id from the case class in order to group on it. Is there a cleaner way to do this that doesn't involve a UDF?

Reply:
Probably you can just `select` the `notificationId` field from `'data`